### ISSUE TITLE
HDR Issue on Stremio Desktop App for Windows 11

### ISSUE THREAD

**Issue by esj006** [Stremio/stremio-bugs#836]
Description:
When using HDR on the Stremio desktop app on Windows 11, the colors appear washed out and not as vivid as expected. This issue persists despite various attempts to adjust settings.

System Information:

Stremio Version: 4.4.168
Web UI Version: 4.4.172
Server Version: 4.20.8
TV: LG OLED 65 C1
GPU: ASUS ROG Strix GeForce RTX 3090
OS: Windows 11 Pro Version 22H2 (OS Build 22631.3737)
Processor: AMD Ryzen 7 5800X 8-Core Processor
RAM: 32 GB
Motherboard: ASUS PRIME X570-P
Steps to Reproduce:

Enable HDR on Windows 11.
Open the Stremio desktop app.
Play any HDR content.
Observed Behavior:
The HDR content appears washed out and lacks vibrancy.

Expected Behavior:
HDR content should appear vivid and with accurate color representation.

Troubleshooting Attempts:

Tried turning HW acceleration on and off.
Adjusted settings in the NVIDIA control panel under "Use NVIDIA color settings":
RGB
YCbCr422
Color depth: 8 bpc and 10 bpc
Dynamic range: Full and Limited
Despite these adjustments, the issue persists with no improvement in color quality.

Additional Information:

HDR works correctly when playing content in an external player, with the HDR content appearing much more vibrant as expected.
This issue is discussed in more detail here: [Reddit post](https://www.reddit.com/r/Stremio/comments/1azqd92/hdr_on_stremio_desktop_app_windows_11_hdr_looking/)

**Comment by Tgo1014** [comment]
Any hope to have HDR in the internal player?

**Comment by esj006** [comment]
I hope the HDR support issue can be fixed soon. Additionally, on Windows, the external player option is only accessible by manually editing the .js file, whereas on other platforms, it is available without any modifications. This feels like a bug, and I hope it can be addressed. Also, when using the external player via the .js file, the progress bar doesn’t retain playback position, which is another issue.

**Comment by guilty-p01nt3r** [comment]
The external player can be used only on android.
On Linux there are the same problems, the quality of the image of the internal player is quite inferior to the external one (MPV).
I would like this feature too.

**Comment by TRtomasz** [comment]
Closing in favor of fixing this issue in stremio 5 new shell version.

**Comment by AnikcuF** [comment]
Issue still present in 5

**Comment by skbtwiz** [comment]
Agreed, issue is still present as of August 2025. Using Windows 11 on an HX370/Radeon 890m laptop with the latest Adrenalin drivers. Internal Video Player of Stremio is not playing anything in actual HDR, as I can move the slider in Windows HDR Settings and the video brightness changes when it is not supposed to. Request this issue be reopened and kept open until a working solution is confirmed to be implemented.

**Comment by TRtomasz** [comment]
@skbtwiz  what do you mean by `not playing in actuall hdr`?

**Comment by AnikcuF** [comment]
![Image](https://github.com/user-attachments/assets/2c57c43b-2983-42c1-bfa0-5b6f970ad9d3)

Native app changes video output to SDR.

![Image](https://github.com/user-attachments/assets/f7012098-0a45-48be-95af-caf79d546d4a)

Web app output is HDR.

VLC launched from either web or native is also HDR.

When playback starts in native, screen temporarily goes black as it switches to SDR, pausing play (while native app still running) blanks again as it returns to HDR.

**Comment by skbtwiz** [comment]
Can confirm, native app outputs SDR while webapp in Edge is outputting HDR properly. I didn't even think to try the webapp, thanks for the suggestion @AnikcuF !

I would have gladly switched to the webapp but I can't seem to figure out how to get rid of horizontal grey bars at the top and bottom in my video playback. Perhaps it has got to do with the fact that I am using a 16:10 display? To me they seem to be part of the Stremio UI. Would love for these to be gone because I am using an OLED display and don't want to risk burn-in.

<img width="2879" height="1799" alt="Image" src="https://github.com/user-attachments/assets/0df15dcd-d68f-4928-8bf2-cd4a24406e4c" />

**Comment by TRtomasz** [comment]
Do you have hdr enabled in your windows settings for that display?

**Comment by AnikcuF** [comment]
> Do you have hdr enabled in your windows settings for that display?

Yes, in the images I posted above, the overlay showing SDR or HDR is produced by the display itself.

**Comment by TRtomasz** [comment]
@AnikcuF there is an option in windows to enable hdr for the display

<img width="1026" height="217" alt="Image" src="https://github.com/user-attachments/assets/597c0436-caaf-4bd8-92a6-6897a0e24b63" />
do you have it enabled?

**Comment by skbtwiz** [comment]
Yes, I think it is pretty obvious this option is enabled if we are getting an HDR output on our displays.

**Comment by AnikcuF** [comment]
> [@AnikcuF](https://github.com/AnikcuF) there is an option in windows to enable hdr for the display
> 
>  do you have it enabled?

Yes

**Comment by TRtomasz** [comment]
Thanks for checking this

**Comment by Oldmangamess** [comment]
I'm still having this issue. I've done everything, and it feels like there is a 10% opacity black filter on everything. It is unwatchable, and this bug has been reported for many months. I have an Odyssey G9, and it just doesn't work when everything else does.

**Comment by TRtomasz** [comment]
To be fixed this require internal mpv update, once they release new version and we update it, this should be fixed.
so we are looking for version newer than 0.40

**Comment by snowmobile2004** [comment]
Looks like MPV 0.41 released 3 weeks ago, perhaps that internal mpv update can be worked on now?
https://github.com/mpv-player/mpv/releases/tag/v0.41.0

**Comment by TRtomasz** [comment]
amazing will check it out @snowmobile2004

**Comment by Danziga1** [comment]
Any updates on this issue? Seems to be long outstanding problem for HDR on the desktop app player.

**Comment by jeregon1** [comment]
For me, HDR10+ content correctly displays in Windows 11 with shell v5.0.16 and app v5.0.0-beta.31

**Comment by skbtwiz** [comment]
I really hope they add a proper "HDR" indicator in the Windows app during playback because I'm literally finding it very hard to tell when its working and when not. I have to open the same show in actual netflix and then in Stremio to compare, and even then I'm not sure.

Such an indicator would be wonderful for diagnosing all such issues.

**Comment by kKaskak** [comment]
any issues present with HDR displaying washed out content on newest 5.0.20 version on windows? i cannot reproduce on my device

**Comment by kKaskak** [comment]
https://github.com/Stremio/stremio-web/pull/1219 related

**Comment by AnikcuF** [comment]
> any issues present with HDR displaying washed out content on newest 5.0.20 version on windows? i cannot reproduce on my device

I haven't noticed it anymore, when I get home I'll test with HDR content to confirm.

**Comment by kKaskak** [comment]
any issues with HDR on windows on the newest shell v5beta? check stremio.com to download

**Comment by AnikcuF** [comment]
Forgot to check, sorry.

Can confirm HDR works:

App 5 beta 36
Server 4.20.17
Shell 5.0.21

**Comment by kKaskak** [comment]
> Forgot to check, sorry.
> 
> Can confirm HDR works:
> 
> App 5 beta 36
> Server 4.20.17
> Shell 5.0.21

Nice
